corehq/apps/app_manager/management/commands/helpers.py

```diff
@@ -56,18 +60,34 @@ def add_arguments(self, parser):
             default=False,
             help="Perform the migration but don't save any changes",
         )
+        parser.add_argument(
+            '--restartable',
+            action='store_true',
+            default=False,
+            help="Uses two txt files to track progress of migrated docs, so that the command can be continue where"
+                 "it left off after being interrupted.",
+        )
```

Comment:
If there's no real disadvantage (I can't think of one), you might just make this _always_ the behavior to simplify things